bin/gftools-push-status.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env python3
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""Check push status of families on Dev, Sandbox and Production servers.
+
+This script will determine whether the families listed in
+to_production.txt and to_sandbox.txt in the google/fonts dir have been
+pushed to their respective servers within an allocated date.
+
+If the files are empty, you can always use git checkout to view past
+states.
+
+
+Usage:
+
+Check push status for the past month:
+gftools push-status path/to/google/fonts
+
+Check push status for specified date:
+gftools push-status path/to/google/fonts -pd 2020-07-01
+"""
+import argparse
+from argparse import RawDescriptionHelpFormatter
+import os
+from pathlib import Path
+import gftools.fonts_public_pb2 as fonts_pb2
+from google.protobuf import text_format
+from datetime import datetime, timedelta
+import json
+import requests
+
+
+DEV_META_URL = "http://fonts-dev.sandbox.google.com/metadata/fonts"
+SANDBOX_META_URL = "http://fonts.sandbox.google.com/metadata/fonts"
+PROD_META_URL = "http://fonts.google.com/metadata/fonts"
```

Comment:
only the host varies, no real reason to define whole thing repeatedly. use https if possible.